lib/components/layouts/MenuBar/Search.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Search as SearchIcon } from "@bigbinary/neeto-icons";
+import { Close } from "@bigbinary/neeto-icons";
+
+import { Input, Button } from "../../index";
+
+const Search = (props) => (
+  <div className="nui-menubar__search">
+    <Input
+      type="search"
+      placeholder="Search"
+      prefix={<SearchIcon size={16} />}
+      {...props}
+    />
+    <Button
+      size="large"
+      style="text"
+      icon={Close}
+      iconSize={20}
+    />
```

Comment:
We need to collapse `Search` on click.

Reply:
I have a doubt regarding the implementation of the collapse. Currently `Search` is an independent component, so an onClick prop to the button would suffice?  So the state and logic for collapsing the search would be taken care of at the place we use the component.

Reply:
@edwinbbu Please share your views regarding this.

Reply:
@ajmaln Let's have another prop in order to collapse the search on click.
@karthiknmenon I am not clear the search field comes on clicking the search icon. There is no reference from the figma.

https://www.loom.com/share/aed76369737a47ac80161a876512c6f7

Reply:
@karthiknmenon How about we change the Search Icon to close Icon on clicking the search. It would be more cleaner.

cc @praveen-murali-ind